x-pack/plugins/cases/common/utils/owner.ts

```diff
@@ -14,7 +14,21 @@ export const isValidOwner = (owner: string): owner is keyof typeof OWNER_INFO =>
 export const getCaseOwnerByAppId = (currentAppId?: string) =>
   Object.values(OWNER_INFO).find((info) => info.appId === currentAppId)?.id;
 
-export const getOwnerFromRuleConsumerProducer = (consumer?: string, producer?: string): Owner => {
+export const getOwnerFromRuleConsumerProducer = ({
+  consumer,
+  producer,
+  isServerlessSecurity,
+}: {
+  consumer?: string;
+  producer?: string;
+  isServerlessSecurity?: boolean;
+}): Owner => {
+  // This is a workaround for a very specific bug with the cases action in serverless security
+  // More info here: https://github.com/elastic/kibana/issues/186270
+  if (isServerlessSecurity) {
```

Comment:
I am curious why we don't need check for o11y.

Reply:
I am also curious. Stack cases are not available in o11y. What is the current behavior if you create a stack alert from the stack management page?

Reply:
I forgot that, thanks!